pkg/lustre-driver/lustre.go

```diff
@@ -28,10 +28,10 @@ import (
 )
 
 func NewLustreDriver() driver.DriverApi {
-	return &lustreDriver{}
+	return &LustreDriver{}
 }
 
-type lustreDriver struct{}
+type LustreDriver struct{}
```

Comment:
Capitalized `lustreDriver` so it can be exported and used outside the package, let me know if this is problematic.

Reply:
Why does this need to be exported?